packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/actions/assistant/message.operation.ts

```diff
@@ -257,7 +306,7 @@ export async function execute(this: IExecuteFunctions, i: number): Promise<INode
 				tools: assistantTools,
 			});
 		}
-		filteredResponse = omit(response, ['signal', 'timeout']) as IDataObject;
+		filteredResponse = omit(response, ['signal', 'timeout', 'content', 'runId']) as IDataObject;
```

Comment:
Why do we need to remove these from the response? Maybe a comment would explain a bit here?

Reply:
Because Langchain returns the original input as `content` which we don't need in n8n. The other ones are just configuration properties, also not relevant to the user